earthfile2llb/interpreter.go

```diff
@@ -403,6 +405,24 @@ func (i *Interpreter) handleForArgs(ctx context.Context, forArgs []string, sl *s
 	return variable, instances, nil
 }
 
+func (i *Interpreter) handleWait(ctx context.Context, waitStmt spec.WaitStatement) error {
+	if !i.converter.ftrs.WaitBlock {
+		return i.errorf(waitStmt.SourceLocation, "the WAIT command is not supported in this version")
+	}
+
+	err := i.converter.pushWaitBlock(ctx)
+	if err != nil {
+		return err
+	}
+
+	err = i.handleBlock(ctx, waitStmt.Body)
+	if err != nil {
+		return err
+	}
+
+	return i.converter.popWaitBlock(ctx)
```

Comment:
Nit: We should probably make `PushWaitBlock` and `PopWaitBlock` public, since they are being called from the interpreter. I know that they are in the same package - it just helps to clarify what the external API of the converter is.